format_cpp.sh

```diff
@@ -8,4 +8,4 @@ find . \
     -not \( -path "*/build/*" -prune \) \
     -not \( -path "*/install/*" -prune \) \
     \( -name *.h -o -name *.hpp -o -name *.c -o -name *.cc -o -name *.cpp \) \
-    | xargs clang-format-5.0 -style=file -i
+    | xargs clang-format-6.0 -style=file -i
```

Comment:
in the future, we might change this to also add [a newline at the end of all files](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline), but let's leave it for another PR